### ISSUE TITLE
[Bug]: The text shadow display on the TextEditor is not correct

### ISSUE THREAD

**Issue by thanglq1** [hm21/pro_image_editor#131]
### Package Version

4.0.8

### Flutter Version

3.22.2

### Platforms

Android, iOS

### How to reproduce?

Hi @hm21. I added a text shadow, but its display on the TextEditor is not correct. I think it must be display the same MainEditor.

https://github.com/hm21/pro_image_editor/assets/26729342/15b681e5-f283-49e6-81e7-fe46ef85408e

### Example code (optional)

```Dart
IconButton(
          tooltip: 'Blue Shadow',
          color: Colors.amber,
          padding: const EdgeInsets.symmetric(horizontal: 8),
          icon: const Text(
            'Blue Shadow',
          ),
          onPressed: () {
            textEditor.setTextStyle(
              textEditor.selectedTextStyle.copyWith(
                shadows: <Shadow>[
                  const Shadow(
                    offset: Offset(1.0, 1.0),
                    blurRadius: 1,
                    color: Colors.blueAccent,
                  ),
                ],
              ),
            );
          },
        ),
```

### Device Model (optional)

iPhone SE (iOS 17.0)

**Comment by hm21** [comment]
Thank you for reporting this issue. I just released version `4.0.10` which should resolve this issue. Please let me know if this issue persists by you after you updated.

**Comment by thanglq1** [comment]
Hi @hm21. Thank you. Its working.
